ferrocene/doc/evaluation-report/src/self-test/tool-analysis.rst

```diff
@@ -0,0 +1,91 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. default-domain:: qualification
+
+Tool Analysis
+=============
+
+Potential Errors
+----------------
+
+.. hazop-error::
+   :id: ERR_SELF_TEST_MISSING_TARGET_CHECKS
+   :caused_by: USE_SELF_TEST_CHECK
+
+   | **Risk:** Invalid output
+   | **Detectable:** YES
+
+   Tool does not perform checks for some installed targets.
+
+.. hazop-error::
+   :id: ERR_SELF_TEST_FALSE_NEGATIVES
+   :caused_by: USE_SELF_TEST_CHECK
+
+   | **Risk:** Invalid output
+   | **Detectable:** NO
+
+   Tool fails to detect a faulty toolchain installation.
+
+.. hazop-error::
+   :id: ERR_SELF_TEST_FALSE_POSITIVES
+   :caused_by: USE_SELF_TEST_CHECK
+
+   | **Risk:** A broken Ferrocene Installation (:id:`RUSTC_ERR_INSTALL_01`)
+   | **Detectable:** YES
+
+   Tool falsely indicates a faulty toolchain installation.
+
+.. hazop-error::
+   :id: ERR_SELF_TEST_MODIFIES_TOOLCHAIN
+   :caused_by: USE_SELF_TEST_CHECK
+
+   | **Risk:** Invalid output
+   | **Detectable:** NO
+
+   Tool creates, changes, or deletes toolchain files/directories.
+
+.. _self-test_avd:
+
+Detection Measures and Usage Restriction
+----------------------------------------
+
+Detection measures and usage restrictions are documented in the :doc:`safety
+manual <safety-manual:self-test/constraints>`.
+
+.. _ferrocene_self_test_iso_tool_classification:
```

Comment:
You could also prefix the links with just "fst", is a bit shorter
```suggestion
.. _fst_iso_tool_classification:
```

Reply:
f6d8fb1f39b instead removing the `ferrocene_` prefix, to avoid needing to know what `fst` is